mlflow/langchain/__init__.py

```diff
@@ -849,7 +850,8 @@ def _load_model_from_local_fs(local_model_path):
         return _load_model_code_path(code_path, config_path)
     else:
         _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
-        return _load_model(local_model_path, flavor_conf)
+        with patch_langchain_type_to_cls_dict():
```

Comment:
Q:

Instead of patching  langchain's `get_type_to_cls_dict`,
can we directly modify code of `mlflow.langchain.runnable._load_model_from_config`?

Reply:
Good point, we should patch here because we have to patch both `_load_base_lcs` (for normal chain) and `_load_model_from_config` (for runnables).

Reply:
got it, this way can only support runnable type langchain model.